src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (a) => {
+    if (a !== undefined) {
```

Comment:
```suggestion
    if (a) {
```

Reply:
But if I'll do this, than with '0' input won't work correctly

Reply:
|| a === 0

Reply:
Then why this better than a !== undefined? :)